ajax/libs/vexflow/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "vexflow",
+  "filename": "vexflow-min.js",
+  "version": "1.2.84",
+  "description": "A JavaScript library for rendering music notation and guitar tablature",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/0xfe/vexflow.git"
+  },
+  "author": {
+    "name": "Mohit Muthanna Cheppudira",
+    "email": "mohit@muthanna.com",
+    "url": "http://muthanna.com"
+  },
+  "license": "MIT",
+  "homepage": "http://vexflow.com",
+  "keywords": [
+    "music",
+    "notation",
+    "guitar",
+    "tablature"
+  ],
+  "npmName": "vexflow",
+  "npmFileMap": [
+    {
+      "basePath": "releases",
+      "files": [
+        "vexflow*"
```

Comment:
Let's modify this to `**/vexflow!(*.html)`. `releases` is similar to `dist`. We usually need most files under them. The pattern I suggest is based on we need all `**/verflow*` except `docs/vexflow_font.html` in v1.2.6 ~ v1.2.83. If you have a better idea. Just do it!